authenticator_session.go

```diff
@@ -8,10 +8,19 @@ import (
 	"github.com/gorilla/sessions"
 	"github.com/pkg/errors"
 	"golang.org/x/oauth2"
-	"k8s.io/apiserver/pkg/authentication/authenticator"
-	"k8s.io/apiserver/pkg/authentication/user"
 )
 
+type Authenticator interface {
+	// Authenticate tries to authenticate a request
+	// returns UserInfo, bool specifying whether the user was found, a
```

Comment:
is this comment complete?

Reply:
ill clear it up a bit and force push again. thanks! :)